lib/gitarro/backend.rb

```diff
@@ -244,19 +244,32 @@ def reviewed_pr?(comm_st, pr)
 
   # this function will check if the PR contains in comment the magic word
   # # for retrigger all the tests.
+  # TODO: To be deprecated when retriggered_by_checkbox? is welcomed by users
   def retriggered_by_comment?(pr_number, context)
     magic_word_trigger = "gitarro rerun #{context} !!!"
     # a pr contain always a comments, cannot be nil
     @client.issue_comments(@repo, pr_number).each do |com|
       # delete comment otherwise it will be retrigger infinetely
       if com.body.include? magic_word_trigger
+        puts "Re-run test \"#{context}\""
         @client.delete_comment(@repo, com.id)
         return true
       end
     end
     false
   end
 
+  # this function will check if the PR contains a checkbox
+  # for retrigger all the tests.
+  def retriggered_by_checkbox?(pr, context)
+    return false if pr.nil?
```

Comment:
i don't think this is needed, a pr object isn't nil otherwise we catch it before.

Why did you add this check here?

Reply:
Totally right.